src/apps/HeyGPT/HeyGPT/Services/CharacterService.cs

```diff
@@ -48,6 +48,34 @@ public void SetSelected(ChatCharacterViewModel? selected)
         Selected = selected ?? DefaultCharacter;
     }
 
+    public ChatCharacterDetails GetCharacter(CharacterType characterType)
+    {
+        var character = Characters.FirstOrDefault(x => x.CharacterType == characterType);
+
+        // Yuck!!!!!
```

Comment:
Look I am not worried about this code right now... It needs to be rewritten to handle loading the characters from disk and a way to add characters etc.    We are focusing on application lifecycle at the moment.